src/style.css

```diff
@@ -1 +1,95 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+body {
+  display: block;
+  box-sizing: border-box;
+}
+
+.search-form,
+.search-form::placeholder {
```

Comment:
The `::placeholder` pseudo-element should be used with the `.search-form__input` class, not `.search-form`. This is because `::placeholder` applies to input elements, not form elements.